tests/functional/provisioning/test_dynamic_provisioning_block.py

```diff
@@ -868,3 +869,52 @@ def test_100gb_block_pvc_create_and_delete_twice(self):
             # Delete the PVC
             oc_delete(self.node, 'pvc', pvc_name)
             wait_for_resource_absence(self.node, 'pvc', pvc_name)
+
+    @pytest.mark.tier2
+    def test_block_provisioner_multiple_custers(self):
+        """Check block provisioner and verify on multiple clusters
+        """
+        # Skip test if registry project is not present
+        self.registry_sc = self.storage_classes.get(
+            'registry_block_storage_class')
+        if not self.registry_sc:
+            self.skipTest(
+                "Config file doesn't have key "
+                "openshift.dynamic_provisioning.storage_classes")
+
+        self._registry_heketi_server_url = self.registry_sc.get('resturl')
+        self._registry_project_name = self.registry_sc.get(
+            'restsecretnamespace')
+        if not (self._registry_heketi_server_url
+                and self._registry_project_name):
+            self.skipTest(
+                "Config file doesn't have key"
+                "'storage_classes.registry_block_storage_class.resturl' or "
+                "'storage_classes.registry_block_storage_class"
+                ".restsecretnamespace'")
+
+        size = 1
+        prefix = 'autotest-pvc-{}'.format(utils.get_random_str(size=5))
+
+        # Create PVC and verify multipath in glusterfs namespace
+        switch_oc_project(self.node, 'glusterfs')
```

Comment:
Add cleanup.
Is it not that glusterfs the default project?

Reply:
Please refer this: https://review.gluster.org/#/c/glusterfs-containers-tests/+/25221/9/tests/functional/provisioning/test_dynamic_provisioning_block.py@890

Reply:
I think its better to remove it, as It's already in the baseclass.

Reply:
I think it's better to remove it, as we are already switching in baseclass

Reply:
and update comment as `Create PVC in default namespace and verify multipath`
`